src/components/ui/BlankPage.tsx

```diff
@@ -0,0 +1,13 @@
+import type React from "react";
+
+const BlankPage: React.FC = () => {
+  return (
+    <div className="flex flex-col items-center justify-center h-full text-discord-text-muted">
+      <div className="text-6xl mb-4">👋</div>
+      <h2 className="text-xl font-semibold">Welcome to your server!</h2>
```

Comment:
Could be a prop and display the name of the server like "Welcome to h4ks" but minor nitpick